osu.Server.Spectator/Services/SharedInterop.cs

```diff
@@ -94,6 +100,8 @@ private async Task<string> runCommand(HttpMethod method, string command, dynamic
             }
             catch (Exception e)
             {
+                bool allowRetry = true;
```

Comment:
@peppy please check that you're ok with this since you [previously objected](https://github.com/ppy/osu-server-spectator/pull/274#discussion_r1977088539) to similar stuff

Reply:
I dunno it reads pretty bad to me but I guess if we have to log this way there's no real avoiding it.